test/config.js

```diff
@@ -220,3 +235,85 @@ captureBodyTests.forEach(function (captureBodyTest) {
     })
   })
 })
+
+test('disableInstrumentation', function (t) {
+  // var hook = require('require-in-the-middle')
```

Comment:
🤔

Reply:
Yep...a bit work-in-progress still. I broke that test and was trying to figure out why. Seems to be some sort of circular dependency issue. I haven't yet figured out a good way to fix it.